content/SCALE/SCALEUIReference/Datasets/EditACLScreens.md

```diff
@@ -51,7 +52,7 @@ Removing this permission results in lost access to the path.
 {{< /hint >}}
 
 ### POSIX ACL Advanced Settings
-The **Advanced** section lets users **Apply Permissions Recursively** to all directories, files, and child datasets within the current dataset.
+The **Advanced** section lets users **Apply Permissions Recursively** to all directories, files, and child datasets within the current dataset, further specifying data sharing capabilities.
```

Comment:
This is about permissions to access the files not sharing them, though in a broader context you can't share what you don't have access too, but. not correct SEO use in this case.
```suggestion
The **Advanced** section lets users **Apply Permissions Recursively** to all directories, files, and child datasets within the current dataset.
```